site/content/integrate/apps/structure/call/_index.md

```diff
@@ -1,136 +1,175 @@
-| `app`                      | App      | (Expansion)                                                                         |
-| `channel`                  | Channel  | (Expansion)                                                                         |
-| `post`                     | Post     | (Expansion)                                                                         |
-| `root_post`                | Post     | (Expansion)                                                                         |
-| `team`                     | Team     | (Expansion)                                                                         |
-| `user`                     | User     | (Expansion)                                                                         |
-| `locale`                   | string   | (Expansion)                                                                         |
-
-### Call response
-
-There are several types ([godoc](https://pkg.go.dev/github.com/mattermost/mattermost-plugin-apps/apps#CallResponseType)) of responses:
-
-| Value      | Description                        |
-|:-----------|:-----------------------------------|
-| `ok`       | OK.                                |
-| `error`    | An error has occurred.             |
-| `form`     | Should open a form.                |
-| `navigate` | Should navigate the user to a URL. |
-
-#### OK response
-
-| Name       | Type   | Description                                                                  |
-|:-----------|:-------|:-----------------------------------------------------------------------------|
-| `type`     | string | Use `ok`.                                                                    |
-| `markdown` | string | (Optional) Markdown text that will be sent to the user as an ephemeral post. |
-
-#### Error response
-
-| Name    | Type   | Description                                                       |
-|:--------|:-------|:------------------------------------------------------------------|
-| `type`  | string | Use `error`.                                                      |
-| `error` | string | Markdown text that will be sent to the user as an ephemeral post. |
-
-#### Form response
-
-| Name   | Type   | Description   |
-|:-------|:-------|:--------------|
-| `type` | string | Use `form`.   |
-| `form` | form   | Form to open. |
-
-#### Navigate response
-
-| Name                   | Type   | Description                                                  |
-|:-----------------------|:-------|:-------------------------------------------------------------|
-| `type`                 | string | Use `navigate`.                                              |
-| `navigate_to_url`      | string | URL to navigate to.                                          |
-| `use_external_browser` | bool   | Whether the navigation link wll open in an external browser. |
-
+A Call defines an App action that can be invoked.
```

Comment:
```suggestion
A Call defines an App action that can be invoked, or a request for App data.
```

Reply:
Webhooks are received by the server which authenticates them, then invokes the app as a Call.